### DISCUSSION TITLE
Plz help me .

### DISCUSSION THREAD

**Discussion by TrongChuongDao** [NationalSecurityAgency/ghidra#4619] [General]
![image](https://user-images.githubusercontent.com/96568661/192240631-e2fcdc53-9d50-44dd-8766-789da2b333ee.png)

**Comment by astrelsky** [comment]
Have you tried giving it what it's asking for?

**Reply by TrongChuongDao** [reply]
I can't setup ghidra ơn ưin 10 . Thanks

**Comment by ryanmkurtz** [comment]
For some reason Ghidra could not create a directory in `c:\Users\Chuong\`.  Are there any permission restrictions on that directory?

**Reply by TrongChuongDao** [reply]
You can guide for me . Thanks

**Reply by TrongChuongDao** [reply]
> For some reason Ghidra could not create a directory in `c:\Users\Chuong\`.  Are there any permission restrictions on that directory?

You can guide for me . Thanks.

**Reply by ryanmkurtz** [reply]
You'll need to provide more information on that directory and your environment.  What version of Window are you using?  Do you have any language packs installed?  Are there unicode characters in your paths?  What version of Java are you using? 

It looks like you have a java installed an it's on your %PATH%.  But for some reason our `LaunchSupport` java program had trouble creating the `.ghidra` directory in your home directory.  This would typically be due to something about your environment.

**Reply by TrongChuongDao** [reply]
> You'll need to provide more information on that directory and your environment. What version of Window are you using? Do you have any language packs installed? Are there unicode characters in your paths? What version of Java are you using?
> 
> It looks like you have a java installed an it's on your %PATH%. But for some reason our `LaunchSupport` java program had trouble creating the `.ghidra` directory in your home directory. This would typically be due to something about your environment.

![Screenshot_1](https://user-images.githubusercontent.com/96568661/192424261-559b7d21-7316-4e20-bb33-50cdecc251bf.png)
![Screenshot_2](https://user-images.githubusercontent.com/96568661/192424270-849212a6-f8ac-4ecd-9ea4-9742626e1b5f.png)

**Comment by graps01** [comment]
Hi TrongChuongDao,
Have you figured out how to link up a JDK with your Ghidra (or other reverse engineering) project setup ? I was able to add a PATH entry to my PC using the Windows environment variable dialog box. From there, I copied the absolute path to my PATH variable and ran the program.
